misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt

```diff
@@ -201,7 +201,7 @@ internal class ZkLease(
     val desiredLeaseOwner = try {
       clusterSnapshot.resourceMapper[leaseResourceName]
     } catch (e: NoMembersAvailableException) {
-      log.warn { e.message }
+      // no healthy members in the cluster yet
```

Comment:
can we instead add `tryResourceMapper` method that returns a nullable owner?

Reply:
sure, I added it to this class, didn't want to change the interface

Reply:
just so I understand this correctly: what does returning null here mean? does that mean the current process is the owner? or it means there is no owner so it tries to grab the lock?

Reply:
It means there is no healthy pod to act as the owner, so no one will try to grab the lock